src/features/auth/components/register-form.tsx

```
@@ -0,0 +1,107 @@
+    const allErrors = Object.values(formState.errors);
+    const hasErrors = allErrors.length > 0;
+
+    console.log(formState.errors)
+
+    return (
+        <div className="flex justify-center h-screen">
+            <div className="flex flex-col p-[20px] pt-[24px] w-[361px]">
+                <div className="flex flex-col">
+                    <span className="text-title-1 mb-3">회원가입</span>
+                    <span className="text-title-3">기본정보</span>
+                    <span className="text-label-1 mb-1 mt-2.5 text-[#3d3d3d]">GIST 이메일</span>
```

ESLint

[error] Replace `····················<span·className="text-label-1·mb-1·mt-2.5·text-[#3d3d3d]">GIST·이메일</span>` with `··········<span·className="text-label-1·mb-1·mt-2.5·text-[#3d3d3d]">⏎············GIST·이메일`

(prettier/prettier)